### PR TITLE
Local control API call

### PR DESCRIPTION
**Pull request by erendrake**
This adds HasLocalControl to the API, i was building an interface for users in kOS to query stuff about remotetech and i realized i had a few places were we were doing the same calculation that RT is for determining local control. 

If RTs definition for that ever changes, kOS would be out of sync so this new endpoint made sense.

### CODE REVIEW

File: src/RemoteTech/API/API.cs [hunk 1]
```diff
@@ -93,37 +102,33 @@ public static double GetSignalDelayToSatellite(Guid a, Guid b)
             return delayBetween;
         }
 
-        public static void ReceiveData(ConfigNode ExternalData) //exposed method called by other mods, passing a ConfigNode to RemoteTech
+        public static void ReceiveData(ConfigNode externalData) //exposed method called by other mods, passing a ConfigNode to RemoteTech
         {
-            if (ExternalData != null) //check we were actually passed a config node
```
Comment (erendrake) [inline src/RemoteTech/API/API.cs]: The rest of this file is a refactor of the current method with no material changes.

File: src/RemoteTech/API/API.cs [hunk 2]
```diff
@@ -93,37 +102,33 @@ public static double GetSignalDelayToSatellite(Guid a, Guid b)
...
-                    RTDebugUnit.print("RT Error: Invalid ConfigNode passed by other mod");
+                    extCmd.vslGUID = vsl2.id;
+                    extCmd.vsl = vsl2;
                 }
+                FlightComputer.FlightComputer fltComp = RTCore.Instance.Satellites[extCmd.vslGUID].FlightComputer;
+                fltComp.Enqueue(extCmd);
+            }
+            catch
+            {
+                RTLog.Notify("Invalid ConfigNode passed by other mod", RTLogLevel.API);
```
Comment (erendrake) [inline src/RemoteTech/API/API.cs]: except for this, we have a logger for this kind of thing already.

File: src/RemoteTech/RTLog.cs [hunk 3]
```diff
@@ -26,17 +26,17 @@ public static class RTLog
 
         static RTLog()
```
Comment (erendrake) [inline src/RemoteTech/RTLog.cs]: all of the changes here were just removing unnecessary qualifiers and fixing casing

File: src/RemoteTech/API/API.cs (hunk 1 above)
Comment (Peppie23) [inline src/RemoteTech/API/API.cs]: yep thats ok. I also want to do a refactor as i mentioned in #233

**Comment by Peppie23** [comment]
Bind to task #349

**Pull request merged**